Katacoda scenarios for modules collection using localstack (#273)
hashicorp/katakoda: terraform-build-modules/step1.md

Open `main.tf`{{open}}. 

This configuration includes three blocks:

- The `"aws" provider` defines your provider. This Katacoda scenario uses 
[localstack](https://localstack.cloud/) to provide a testing environment with 

> same functionality and APIs as the real AWS cloud environment. This way you can 

Author: I changed it to "similar". localstack does a pretty good job as a mocking framework

> - The `"aws" provider` defines your provider. This Katacoda scenario uses [localstack](https://localstack.cloud/) to provide a 
> testing environment on your local machine that provides the same functionality 
> and APIs as the real AWS cloud environment. This way you can learn how to organize
> your Terraform configuration without needing an AWS account.

Reviewer: ```suggestion
- The `"aws" provider` defines your provider. This Katacoda scenario uses [localstack](https://localstack.cloud/) to provide a 
testing environment with same functionality 
and APIs as the real AWS cloud environment. This way you can learn how to organize
your Terraform configuration without needing an AWS account.
```
